src/riak_core_security.erl

```diff
@@ -82,6 +98,28 @@
 -type bucket() :: {binary(), binary()} | binary().
 -type permission() :: {string()} | {string(), bucket()}.
 -type userlist() :: all | [string()].
+-type username() :: string().
+-type metadata_key() :: string().
+-type metadata_value() :: term().
+-type options() :: [{metadata_key(), metadata_value()}].
+
+-spec find_by_metadata(metadata_key(), metadata_value()) -> [{username(), options()}].
+find_by_metadata(Key, Value) ->
+    riak_core_metadata:fold(
+      fun({_Username, [?TOMBSTONE]}, Acc) ->
```

Comment:
Rejecting tombstones seems pretty common in this file. We just copied and pasted it, but I'd like to refactor it into a named function.